srcpkgs/PolyMC/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'PolyMC'
+pkgname=PolyMC
+version=1.1.0
+revision=1
+_nbtversion="dc72a20b7efd304d12af2025223fad07b4b78464"
+_quazipversion="09ec1d10c6d627f895109b21728dda000cbfa7d1"
+build_style=cmake
+configure_args="-DLauncher_BUILD_PLATFORM=Void -DLauncher_PORTABLE=0
+ -DLauncher_VERSION_BUILD=${revision}"
+hostmakedepends="openjdk11 qt5-host-tools qt5-qmake"
+makedepends="qt5-devel"
+depends="virtual?java-runtime"
+short_desc="Custom launcher for Minecraft"
+maintainer="Philipp David <pd@3b.pm>"
+license="GPL-3.0-or-later"
+homepage="https://polymc.org/"
+distfiles="https://github.com/PolyMC/PolyMC/archive/${version}.tar.gz
+ https://github.com/MultiMC/libnbtplusplus/archive/${_nbtversion}.tar.gz
+ https://github.com/stachenov/quazip/archive/${_quazipversion}.tar.gz"
+checksum="4114b11a47a5380c811df890310578ddb3457f0a96caf019f388e1f551d58e99
+ 2ed9dc824007b4f7b12f807f56c98e1248b8a7aba7eca4838085092958dc1595
+ 618700e4e5f1f884e22bc8ed2c2c102305711ebf5a39296bf4c6017a4e6d4e35"
+
+case "$XBPS_TARGET_MACHINE" in
+	armv*) broken="https://github.com/MultiMC/MultiMC5/issues/2895";;
```

Comment:
This is a bad issue to list as justification.

We a) either disable builds for everything except `x86_64` (and maybe `i686`) or b) keep them for non `x86_64` and keep it as an exercise for the user to replace the native x86_64 libs in Minecraft with usable ones for their system.

Reply:
The issue states how MultiMC compilation itself fails for armv* with gcc. It still does to this day, even with PolyMC. I will remove it if/when toolchain updates fix it.